src/Http/Controllers/ServerNotificationController.php

```diff
@@ -44,8 +44,9 @@ public function google(GoogleDeveloperNotificationRequest $request)
 
     /**
      * @param AppStoreServerNotificationRequest $request
+     * public function apple(AppStoreServerNotificationRequest $request)
      */
-    public function apple(AppStoreServerNotificationRequest $request)
+    public function apple(Request $request) // bypass parameter rules
```

Comment:
@DustinSRamirez 

Why didn't you update the rules instead of replacing the request class?